gamestonk_terminal/common/technical_analysis/volume_view.py

```diff
@@ -16,7 +17,7 @@
 
 def plot_ad(
     s_ticker: str, s_interval: str, df_stock: pd.DataFrame, use_open: bool, export: str
-):
+) -> pd.DataFrame:
```

Comment:
This is not the case. this function is not returning any dataframe, but plotting it